### PR TITLE
OBSDOCS-152: Restructure support section to remove duplication and improve findability

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.10+

Issue:
Related to https://issues.redhat.com/browse/OBSDOCS-152

Previously, support information was buried alongside architecture information for different versions of logging, and the modules and content were reused / duplicated unnecessarily. This PR moves this information to a new, more findable top-level support section.

Link to docs preview:
- https://63952--docspreview.netlify.app/openshift-enterprise/latest/logging/cluster-logging-support
- https://63952--docspreview.netlify.app/openshift-enterprise/latest/logging/v5_7/logging-5-7-architecture (content removal)
- https://63952--docspreview.netlify.app/openshift-enterprise/latest/logging/v5_6/logging-5-6-architecture (content removal)
- https://63952--docspreview.netlify.app/openshift-enterprise/latest/logging/v5_5/logging-5-5-architecture (content removal)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: logging/v5_7/logging-5-7-architecture.adoc [hunk 1]
```diff
@@ -8,7 +8,4 @@ toc::[]
 
 :context: logging-5-7-architecture-overview
 include::modules/logging-architecture-overview.adoc[lines=9..31]
-
-include::modules/logging-support-considerations.adoc[leveloffset=+1]
-
 include::snippets/logging-5.7-outputs-snip.adoc[]
```
Comment (libander) [inline logging/v5_7/logging-5-7-architecture.adoc:11]: This isn't displaying correctly in the preview file.

Comment (abrennan89) [reply logging/v5_7/logging-5-7-architecture.adoc:13]: Fixed

**Review by libander — APPROVED** [review]
/lgtm

**Pull request merged**